### PR TITLE
:arrow_up: Update packages for RC1. Closes #487

### LINKED ISSUE #487: ASP.NET 5 RC1 final package versions
This bug is tracking the final versions of the packages.
- ASP.NET 5 packages: 1.0.0-rc1-final
- DNX Class Project:
```html
    "dotnet5.4": {
      "dependencies": {
        "Microsoft.CSharp": "4.0.1-beta-23516",
        "System.Collections": "4.0.11-beta-23516",
        "System.Linq": "4.0.1-beta-23516",
        "System.Runtime": "4.0.21-beta-23516",
        "System.Threading": "4.0.11-beta-23516"
      }
```
- DNX Console Project:
```html
    "dnxcore50": {
      "dependencies": {
        "Microsoft.CSharp": "4.0.1-beta-23516",
        "System.Collections": "4.0.11-beta-23516",
        "System.Console": "4.0.0-beta-23516",
        "System.Linq": "4.0.1-beta-23516",
        "System.Threading": "4.0.11-beta-23516"
      }
    }
```

### PR DESCRIPTION
**Pull request by peterblazejewicz**
A late update to projects packages version for use with RC1 as asked here:
https://github.com/OmniSharp/generator-aspnet/issues/487#issuecomment-157477881

This PR updates:
* ClassLibrary
* ConsoleApplication
* EmptyApplication
* WebAPIApplication
* WebApplication
* WebApplicationBasic

All were tested with `restore`, `build and `dnx web` (where applicabble) to detect possible side change (I haven't spotted one).
Test OS:
```
Microsoft .NET Development Utility
 Version:      1.0.0-rc1-16147
 Type:         Mono
 Architecture: x64
 OS Name:      Darwin
 OS Version:   10.10
 Runtime Id:   osx.10.10-x64
```

/cc
@sayedihashimi

### CODE REVIEW

File: templates/projects/web/project.json [hunk 1]
```diff
@@ -12,23 +12,23 @@
     "EntityFramework.Commands": "7.0.0-rc1-*",
```
Comment (rustd) [inline templates/projects/web/project.json]: All package versions should be rc1-final

File: templates/projects/webapi/project.json [hunk 2]
```diff
@@ -8,14 +8,14 @@
   },
 
   "dependencies": {
-    "Microsoft.AspNet.IISPlatformHandler": "1.0.0-rc1-*",
+    "Microsoft.AspNet.IISPlatformHandler": "1.0.0-rc1-final",
     "Microsoft.AspNet.Mvc": "6.0.0-rc1-*",
```
Comment (rustd) [inline templates/projects/webapi/project.json]: All package versions should be rc1-final

File: templates/projects/webbasic/project.json [hunk 3]
```diff
@@ -8,18 +8,18 @@
   },
 
   "dependencies": {
-    "Microsoft.AspNet.Diagnostics": "1.0.0-rc1-*",
-    "Microsoft.AspNet.IISPlatformHandler": "1.0.0-rc1-*",
+    "Microsoft.AspNet.Diagnostics": "1.0.0-rc1-final",
+    "Microsoft.AspNet.IISPlatformHandler": "1.0.0-rc1-final",
     "Microsoft.AspNet.Mvc": "6.0.0-rc1-*",
```
Comment (rustd) [inline templates/projects/webbasic/project.json]: All package versions should be rc1-final

**Comment by peterblazejewicz** [comment]
@rustd 
OK, I see, like on RC1 branch on home (not yet merged):
https://github.com/aspnet/Home/blob/glennc/rc1/samples/1.0.0-rc1/HelloMvc/project.json#L12-L16

**Pull request merged**